libs/auth/README.md

```diff
@@ -0,0 +1,11 @@
+# auth
```

Comment:
We can populate these READMEs and then render them on our docs via linking, up to you